extensions/blocks/google-calendar/editor.scss

```diff
@@ -16,7 +16,14 @@
 	}
 
 	&-embed-form-editor {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		flex-wrap: wrap;
```

Comment:
Is this bit necessary? The element you're targeting seems to already get these styles.

When I add only the two styles below (lines 25–26) to the textarea, it seems to work as intended.

Reply:
Good catch, I can see these being set elsewhere too:

![Screenshot 2020-10-19 at 09 59 49](https://user-images.githubusercontent.com/411945/96417847-30000e80-11f2-11eb-918b-1fe3baf20f6b.png)

I wasn't expecting the wrapper to already be `flex`.